### DISCUSSION TITLE
How to add links to images?

### DISCUSSION THREAD

**Discussion by nakaokanfy** [Milkdown/milkdown#560] [Q&A]
As below.  🙇 

```md
[![](https://milkdown.dev/polar.jpeg)](https://milkdown.dev/online-demo)
```

If it could be, I'm very happy.
May be related to
- https://github.com/Saul-Mirone/milkdown/issues/555

**Comment by Saul-Mirone** [comment]
Yeah, they're related. I'll figure out a way to support it.

**Reply by Saul-Mirone** [reply]
This is fixed now, will be released in 6.2

**Reply by nakaokanfy** [reply]
@Saul-Mirone Wow, thank you!!!!! 🙇
